app/io/pathfinder/websockets/controllers/WebSocketCrudController.scala

```diff
@@ -24,10 +24,10 @@ abstract class WebSocketCrudController[V <: ebean.Model](
         Logger.info(s"Received web socket crud request $webSocketMessage")
         webSocketMessage match {
             case Update(m, id, value) => Some(Updated(model, writes.writes(dao.update(id, resources.reads(value).get).get)))
-            case Create(m, value) => { Some(Created(model, writes.writes(dao.create(resources.reads(value).get).get))) }
+            case Create(m, value) => Some(Created(model, writes.writes(dao.create(resources.reads(value).get).get)))
             case Delete(m, id) => Some(Deleted(model, writes.writes(dao.delete(id).get)))
             case Read(m, id) => Some(Model(model, writes.writes(dao.read(id).get)))
-            case x => None
+            case x: WebSocketMessage => Some(Error("No support for message: " + WebSocketMessage.format.writes(x) + " for model: "+model.toString))
```

Comment:
This never gets used: The server errors on a None.get and returns nothing. Please fix ASAP. See #19